src/core/utils/common.tsx

```diff
@@ -60,3 +60,19 @@ export const getContainer = (containerId: string, ownerDocument = document) => {
   }
   return container;
 };
+
```

Comment:
Maybe change to `getDocument`
![image](https://user-images.githubusercontent.com/36186912/121160975-3b717a00-c855-11eb-874b-9aa375a00209.png)

Reply:
added as default parameter value